snowplow_tracker/tracker.py

```diff
@@ -185,7 +187,7 @@ def track_page_view(
         context: Optional[List[SelfDescribingJson]] = None,
         tstamp: Optional[float] = None,
         event_subject: Optional[_subject.Subject] = None,
-    ) -> "Tracker":
+    ) -> str:
```

Comment:
I'm not sure it is worth changing the return type and also deprecating these functions as we introduce the event object at the same time.

I would suggest instead to keep these functions without change and only update the return type of the `track()` function. Then make the `track()` function accept an `Event` object instead of just `Payload` and deprecate the `track_xxx()` functions.